types/src/testing.rs

```diff
@@ -83,6 +83,11 @@ impl TestRng {
             *flag.borrow_mut() = true;
         });
     }
+
+    /// Returns the current `seed`.
+    pub fn seed(&self) -> Seed {
+        self.seed
+    }
```

Comment:
Not a blocker, but I don't think we need this getter - the `Display` and `Debug` impls of `TestRng` already print the seed.

Reply:
Yes, good catch, it works indeed:
```
Running 'dont_upgrade_without_switch_block' test with rng=TestRng seed: fab95057ffe9475529432a90bbab216a
```

Fixed in 48e994aa86bd00c201281bc08a19fb3f4c22f642